### PR TITLE
Issue 365 npm audit fix

### PR DESCRIPTION
**Pull request by akuma1**
Resolves: #365 

### PR Safety Checklist:

 - [ ] Added the task to the appropriate release doc under **Enhancements** or **Bug Fixes**
 - [ ] Bump `package.json` & `package-lock.json` version numbers to appropriate release
 - [ ] (optional) All external API changes have been documented
 - [ ] (optional) Build docs: `npm run docs`

### CODE REVIEW

**Review by gabe647 — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -151,24 +150,24 @@
     "@babel/preset-react": "~7.7.0",
     "@babel/runtime": "~7.7.2",
     "@hot-loader/react-dom": "~16.13.0",
-    "@testing-library/dom": "~7.0.4",
+    "@seneca/vorpal": "^2.1.1",
```
Comment (stazrad) [inline package.json:153]: why the new vorpal dep here?

Comment (akuma1) [reply package.json:153]: original Vorpal dependency was last published to npm 5 years back and that has many security vulnerabilities including a critical security vulnerability. This forked version of Vorpal addresses those security vulnerabilities. Our long term plan is to completely re-write our cli to use some modern cli library like @boost/cli.

Comment (stazrad) [reply package.json:153]: 👌

**Review by stazrad — APPROVED** [review]

**Pull request merged**